src/test/java/configs/karate-config.js

```diff
@@ -0,0 +1,18 @@
+function fn() {
```

Comment:
Add a folder for payloads with an example payload

Reply:
Added a folder and sample json